src/tabs/interfaces.ts

```diff
@@ -11,6 +12,14 @@ export interface TabsProps extends BaseComponentProps {
    * - `label` (ReactNode) - Tab label shown in the UI.
    * - `content` (ReactNode) - (Optional) Tab content to render in the container.
    * - `disabled` (boolean) - (Optional) Specifies if this tab is disabled.
+   * - `dismissible` (boolean) - (Optional) Determines whether the tab includes a dismiss icon button. By default, the dismiss button is not included.
+   * - `dismissLabel` (boolean) - (Optional) Specifies an aria-label for the dismiss icon button.
+   *    When a user clicks on this button the onDismiss handler is called.
+   * - `dismissDisabled` (boolean) - (Optional) Determines whether the dismissible button is disabled. We recommend using this only when the dismissible is present
```

Comment:
I don't think we need the bit about recommendation: I think it's pretty clear that it's only relevant when the dismiss button is present. And suggest "dismiss button" for consistency.
```suggestion
   * - `dismissDisabled` (boolean) - (Optional) Determines whether the dismiss button is disabled.
```

Reply:
That makes sense -  good catch on the wording for `dismiss`